### ISSUE TITLE
HTTP Task running multiple times

### ISSUE THREAD

**Issue by alexandrearduini** [Netflix/conductor#386]
Hi,
My HTTP task is experiencing unwanted behavior. When the service takes too long to respond, the task is being called again. I would like to set up the task and workflow to run the task only once regardless of how long the service took to respond to the request.

I configured the task like this:
```
[
    {
    "name": "t",
    "retryCount": 0,
    "timeoutSeconds": 0,
    "timeoutPolicy": "TIME_OUT_WF",
    "retryLogic": "FIXED",
    "retryDelaySeconds": 0,
    "responseTimeoutSeconds": 0
    }
]
```
The workflow is configured like this:
```
{
    "name": "w",
    "version": 1,
    "schemaVersion": 2,
    "inputParameters": [],
    "tasks": [{
            "name": "t",
            "taskReferenceName": "t",
            "type": "HTTP",
            "inputParameters": {
                "http_request": {
                    "uri": "http://localhost:8084/test",
                    "method": "GET"
                }
            }
        }
    ]
}
```
When it is delayed to respond the task is not marked as TIMED_OUT, but the GET method is called several times, and the Poll Count is incremented.
I noticed that at some point the task is queued again (SystemTaskWorkerCoordinator) and then executed, and a new worker is created to execute this task. But I couldn't find a way to set a timeout parameter for it or configure to poll the task only once.

Is there a way to do this?
Thanks!

**Comment by alexandrearduini** [comment]
This is the log when de http_request takes time to respond

```
25409 [system-task-worker-0] INFO  com.netflix.conductor.core.execution.WorkflowExecutor  - Executing HTTP/09770016-9cbd-4b2a-af8f-52fa45b58091-SCHEDULED
129569 [system-task-worker-1] INFO  com.netflix.conductor.core.execution.WorkflowExecutor  - Executing HTTP/09770016-9cbd-4b2a-af8f-52fa45b58091-SCHEDULED
249736 [system-task-worker-2] INFO  com.netflix.conductor.core.execution.WorkflowExecutor  - Executing HTTP/09770016-9cbd-4b2a-af8f-52fa45b58091-SCHEDULED
```

**Comment by kishorebanala** [comment]
While Conductor executes the task only once, the underlying Client Handler (say Jersey Client) executing Http task might have been configured differently. I.e Jersey Client might have default config to retry 3 times if a request times out.

Closing this issue, as there is no activity for a while. Please feel free to open another issue if you still have questions.
